docs/_sass/_main.scss

```diff
@@ -2,6 +2,16 @@
 @import 'colors';
 @import 'fonts';
 
+$color-appBG: $color-green-appBG;
+$color-highlightBG: $color-green-highlightBG;
+$color-borders: $color-green-borders;
+$color-icons: $color-green-icons;
+$color-blue: $color-blue200;
```

Comment:
This should be $color-link

Reply:
color link is other color that defind below

Reply:
Yeah apologies, where else do we use color-blue besides for links?

Reply:
@shawnborton We also use color-blue in hr tag

Reply:
Hmm let me try to restate this. 

Let's get rid of $color-blue from main.scss. "Blue" describes a color, and that would exist in the colors.css file and not in main.scss

Instead, let's just have:

```
$color-link: $color-blue200;
$color-link-hovered: $color-blue100;
```

Reply:
Furthermore, I don't think we should use blue for HRs, we should just use color-border